### DISCUSSION TITLE
mysql为什么没有事务支持？

### DISCUSSION THREAD

**Discussion by stlnkm** [cloudwu/skynet#1149] [Q&A]
如题，建议通过存储过程实现？

**Comment by cloudwu** [comment]
基本原则是，谁有需求谁来添加，欢迎 pr 。

**Comment by cloudfreexiao** [comment]
看 测试用例 有 支持 存储过程？
https://github.com/cloudwu/skynet/blob/master/test/testmysql.lua#L105
